### DISCUSSION TITLE
Typescript sources is not compiled yet 에러 ㅠㅠㅠㅠ

### DISCUSSION THREAD

**Discussion by yj098** [naverz/zepeto-studio-kor#1068] [5. Q&A]
새벽에 멀티플레이 서버 만들기 강좌보고 따라했는데 뭔가 잘못됐는지... 이런 에러가 뜨네요 ㅠㅠㅠ

지금 에러 해결법 계속 찾아보고 있는 중인데 해결이 안되네요 ㅠㅠㅠ
![image](https://user-images.githubusercontent.com/99742984/180865818-60116bec-313f-4ab8-b3ed-208b4ffea939.png)

혹시 이 상태면 타입스크립트 서버 꺼진 건가요??

ㅠㅠㅠㅠㅠㅠㅠㅠㅠ 
이건 어떻게 다시 키나요....?ㅠㅠㅠ
도움 부탁 드립니다... ㅠㅠㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, TS아이콘에 불이 켜져 있으면 타입스크립트 서버가 켜져있는 상황입니다.
만약 서버가 꺼져있다면 월드를 최신 버전으로 업데이트 하시고, 
유니티를 껐다가 다시 켜시면 정상적으로 동작할 수 있습니다.
ASSETS > Reimport All 또는 ZEPETO - Script - All Rebuild
을 하신 후 유니티 재시작을 해보시는 것도 방법입니다.

서버가 동작할때 Typescript sources is not compiled yet 오류가 발생한다면 
주로 스크립트 내에 문법적 오류가 있거나 정의되지 않은 내용이 있을 때 발생합니다.

**Reply by yj098** [reply]
한 번 시도해보겠습니다 :) :) 도움 주셔서 감사합니다 ㅠㅠㅠㅠ

**Reply by chlguswls32** [reply]
yj098님 안녕하세요. 
질문이 있어서 댓글남깁니다. 제페토 멀티플레이 서버 만들기강좌는 어디서 볼 수 있나요?

**Reply by yj098** [reply]
https://www.youtube.com/watch?v=FmK6JNlSjJA&t=58s
이제 봤네여 ㅎㅠㅎ
제페토 스튜디오 가이드에 멀티플레이 강좌가 추가되어 있어서 따라해보고 있습니다 ㅎ
